docs/notebooks/06_differential_expression.ipynb

```diff
@@ -0,0 +1,1306 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    dc.plot_psbulk_samples(pdata, groupby=['sample','condition'], figsize=(13,5))
</pre><p>maybe better to color by dataset? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>